debian/frr.logrotate

```diff
@@ -17,7 +17,7 @@
             # open, as well as the daemons, so always signal the daemons.
             # It's safe, a NOP if (only) syslog is being used.
             for i in babeld bgpd eigrpd isisd ldpd nhrpd ospf6d ospfd \
-                pimd ripd ripngd zebra staticd fabricd; do
+                pimd ripd ripngd zebra pbrd staticd bfdd fabricd vrrpd; do
```

Comment:
I don't know if that matters much, but it would cleaner to have this list ordered the same way as the structure we have in the repo (alphabetic order): `babeld, bfdd, bgpd, ...`

Reply:
that would need to be done in a separate commit. in any case im fine with the changes as is